tensorboard/webapp/feature_flag/views/feature_flag_page_component.ts

```diff
@@ -12,20 +12,48 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
-import {Component, Input} from '@angular/core';
-import {FeatureFlagState} from '../store/feature_flag_types';
+import {Component, EventEmitter, Input, Output} from '@angular/core';
+import {FeatureFlagType} from '../store/feature_flag_metadata';
+import {FeatureFlags} from '../types';
+import {FeatureFlagStatus, FeatureFlagStatusEvent} from './types';
+
 @Component({
   selector: 'feature-flag-page-component',
+  styleUrls: ['feature_flag_page_style.css'],
   templateUrl: `feature_flag_page.ng.html`,
```

Comment:
style and template files should be named feature_flag_page_component.css and feature_flag_page_component.ng.html. "style" is redundant (in our opinion). we include "component" because it differentiates them from possible "container" css and html files, too.